test/rotation_matrix_test.cpp

```diff
@@ -0,0 +1,136 @@
+#include "Eigen/Eigen"
+#include "surface_perception/shape_extraction.h"
+
+#include "pcl/ModelCoefficients.h"
+
+#include <gtest/gtest.h>
+
+namespace surface_perception {
+// Consant dimensions
+const double kLongSide = 2.0;
+const double kShortSide = 1.0;
+
+TEST(TestStandardizeBoxOrientation, IdentityMatrix) {
+  Eigen::Matrix3f expected_matrix = Eigen::Matrix3f::Identity();
+  Eigen::Matrix3f input_matrix = Eigen::Matrix3f::Identity();
+
+  double x_dim, y_dim;
+  Eigen::Matrix3f actual_matrix = StandardizeBoxOrientation(
+      input_matrix, kShortSide, kLongSide, &x_dim, &y_dim);
+
+  ASSERT_TRUE(expected_matrix.isApprox(actual_matrix, 0.0001));
+}
+
+TEST(TestStandardizeBoxOrientation, IdentityMatrixRotate180DegreesAroundYAxis) {
+  Eigen::Matrix3f expected_matrix;
+  expected_matrix << 1.0, 0.0, 0.0, 0.0, -1.0, 0.0, 0.0, 0.0, -1.0;
```

Comment:
Would be nice to align the numbers so that it looks like a 3x3 matrix. If clang-format is giving you problems, you can disable formatting for a block of code: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#disabling-formatting-on-a-piece-of-code

Reply:
Completed